### PR TITLE
New package: gcsf

### PR DESCRIPTION
**Pull request by benalb**

### CODE REVIEW

File: srcpkgs/gcsf/template [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Template file for 'gcsf'
+pkgname=gcsf
+version=0.1.17
+revision=1
+hostmakedepends="cargo rust pkg-config"
```
Comment (cr6git) [inline srcpkgs/gcsf/template]: `hostmakedepends="cargo pkg-config"`

File: srcpkgs/gcsf/template [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+makedepends="fuse-devel libressl-devel"
+short_desc="A FUSE file system based on Google Drive"
```
Comment (cr6git) [inline srcpkgs/gcsf/template]: `short_desc="FUSE file system based on Google Drive"`

File: srcpkgs/gcsf/template [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+	cargo build --release
+}
+
+do_check() {
+	cargo test --release
+}
+
+do_install() {
+	vbin target/release/"${pkgname}"
+
```
Comment (cr6git) [inline srcpkgs/gcsf/template]: you can remove this blank line...

File: srcpkgs/gcsf/template (hunk 1 above)
Comment (benalb) [reply srcpkgs/gcsf/template]: I guess cargo pulls the rust dependency. Will change.

File: srcpkgs/gcsf/template (hunk 2 above)
Comment (benalb) [reply srcpkgs/gcsf/template]: Will change.

File: srcpkgs/gcsf/template (hunk 3 above)
Comment (benalb) [reply srcpkgs/gcsf/template]: Ok, will do.

File: srcpkgs/gcsf/template (hunk 1 above)
Comment (cr6git) [reply srcpkgs/gcsf/template]: Yes, look at the template: `depends="rust"`
https://github.com/void-linux/void-packages/blob/master/srcpkgs/cargo/template#L7

**Pull request merged**